### ISSUE TITLE
bilibili 下载失败

### ISSUE THREAD

**Issue by lovecn** [soimort/you-get#2022]
环境 win7 Python3.5 
```js
$ pip install --upgrade you-get
Collecting you-get
  Downloading you_get-0.4.750-py3-none-any.whl (170kB)
    42% |█████████████▌                  | 71kB 113kB/s
    48% |███████████████▍                | 81kB 120kB
    54% |█████████████████▎              | 92kB 135
    60% |███████████████████▎            | 102kB
    66% |█████████████████████▏          | 112k
    72% |███████████████████████         | 122
    78% |█████████████████████████       | 1
    84% |███████████████████████████     |
    90% |████████████████████████████▉
    96% |██████████████████████████████
    100% |██████████████████████████████
B 216kB/s
Installing collected packages: you-get
  Found existing installation: you-get 0.4.555
    Uninstalling you-get-0.4.555:
      Successfully uninstalled you-get-0.4.555
Successfully installed you-get-0.4.750

$ you-get.exe http://www.bilibili.com/video/av10788059
you-get: [error] oops, something went wrong.
you-get: don't panic, c'est la vie. please try the following steps:
you-get:   (1) Rule out any network problem.
you-get:   (2) Make sure you-get is up-to-date.
you-get:   (3) Check if the issue is already known, on
you-get:         https://github.com/soimort/you-get/wiki/Known-Bugs
you-get:         https://github.com/soimort/you-get/issues
you-get:   (4) Run the command with '--debug' option,
you-get:       and report this issue with the full output.

$ you-get.exe http://www.bilibili.com/video/av10788059 --debug
[DEBUG] get_content: http://www.bilibili.com/video/av10788059
[DEBUG] get_content: http://interface.bilibili.com/playurl?&cid=17800751&from=mi
niplay&player=1&sign=294ad5d2a2601d860effbfec818cd6be
[DEBUG] url_info: http://ws.acgvideo.com/9/76/17800751-1.flv?wsTime=1495733746&p
latform=pc&wsSecret2=188cd5d81500233cc712111758429fe8&oi=1898737553&rate=1280
you-get: version 0.4.750, a tiny downloader that scrapes the web.
you-get: ['http://www.bilibili.com/video/av10788059']
Traceback (most recent call last):
  File "d:\python3\lib\runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "d:\python3\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "d:\python3\Scripts\you-get.exe\__main__.py", line 9, in <module>
  File "d:\python3\lib\site-packages\you_get\__main__.py", line 92, in main
    main(**kwargs)
  File "d:\python3\lib\site-packages\you_get\common.py", line 1444, in main
    script_main('you-get', any_download, any_download_playlist, **kwargs)
  File "d:\python3\lib\site-packages\you_get\common.py", line 1355, in script_ma
in
    download_main(download, download_playlist, args, playlist, output_dir=output
_dir, merge=merge, info_only=info_only, json_output=json_output, caption=caption
)
  File "d:\python3\lib\site-packages\you_get\common.py", line 1157, in download_
main
    download(url, **kwargs)
  File "d:\python3\lib\site-packages\you_get\common.py", line 1437, in any_downl
oad
    m.download(url, **kwargs)
  File "d:\python3\lib\site-packages\you_get\extractors\bilibili.py", line 187,
in bilibili_download
    info_only=info_only)
  File "d:\python3\lib\site-packages\you_get\extractors\bilibili.py", line 101,
in bilibili_download_by_cid
    _, type_, temp = url_info(url, headers={'Referer': 'http://www.bilibili.com/
'})
  File "d:\python3\lib\site-packages\you_get\common.py", line 420, in url_info
    response = urlopen_with_retry(request.Request(url, headers=headers))
  File "d:\python3\lib\site-packages\you_get\common.py", line 307, in urlopen_wi
th_retry
    return request.urlopen(*args, **kwargs)
  File "d:\python3\lib\urllib\request.py", line 163, in urlopen
    return opener.open(url, data, timeout)
  File "d:\python3\lib\urllib\request.py", line 472, in open
    response = meth(req, response)
  File "d:\python3\lib\urllib\request.py", line 582, in http_response
    'http', request, response, code, msg, hdrs)
  File "d:\python3\lib\urllib\request.py", line 504, in error
    result = self._call_chain(*args)
  File "d:\python3\lib\urllib\request.py", line 444, in _call_chain
    result = func(*args)
  File "d:\python3\lib\urllib\request.py", line 696, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "d:\python3\lib\urllib\request.py", line 472, in open
    response = meth(req, response)
  File "d:\python3\lib\urllib\request.py", line 582, in http_response
    'http', request, response, code, msg, hdrs)
  File "d:\python3\lib\urllib\request.py", line 510, in error
    return self._call_chain(*args)
  File "d:\python3\lib\urllib\request.py", line 444, in _call_chain
    result = func(*args)
  File "d:\python3\lib\urllib\request.py", line 590, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden
```

**Comment by rosynirvana** [comment]
#2018
#2021

**Comment by soimort** [comment]
Closed via #1698.

**Comment by lovecn** [comment]
thks, it is ok now
`pip install -U you-get`
